backend/app/views/backend/about_sections/_about_sections.html.erb

```diff
@@ -1,14 +1,28 @@
-<h2>About sections</h2>
+<h2 class="text -main-title -light">About sections</h2>
 
-<% if can?(:create, AboutSection) %>
-  <%= link_to "New", new_about_section_path %>
-<% end %>
+<div class="buttons-bar">
+  <% if can?(:create, Activity) %>
```

Comment:
The method `can?` is checking if the user is authorized to do a particular action. In this case because we are in the AboutSections it should be for the "AboutSection" model, and not activity.